lisa/operating_system.py

```diff
@@ -129,10 +129,11 @@ class OperatingSystem:
 
     __posix_factory: Optional[Factory[Any]] = None
 
-    def __init__(self, node: "Node", is_posix: bool) -> None:
+    def __init__(self, node: "Node", is_posix: bool, is_bmc: bool = False) -> None:
         super().__init__()
         self._node: Node = node
         self._is_posix = is_posix
+        self.is_bmc = is_bmc
```

Comment:
Does it conflict with posix or not? If it doesn't conflict, it means there are 4 combinations with posix. So Does all of them supported?